carbonmark/components/pages/Retire/RetireModal/styles.ts

```diff
@@ -0,0 +1,181 @@
+import { css } from "@emotion/css";
+import breakpoints from "@klimadao/lib/theme/breakpoints";
+import * as typography from "@klimadao/lib/theme/typography";
+
+export const container = css`
+  display: grid;
+  gap: 2rem;
+  background-color: white;
+  border-radius: 1.2rem;
+`;
+
+export const contentContainer = css`
+  display: grid;
+  gap: 2rem;
+
+  &.success {
+    opacity: 0.3;
+  }
+`;
+
+export const formatParagraph = css`
+  p {
+    margin-bottom: 1em;
+    font-family: "DM Sans";
+  }
+`;
+
+export const details = css`
+  font-family: "DM Sans";
```

Comment:
Just wondering about the font-style being addressed here directly.
We do have a typo-system for each variant of font.
For text there is the `Text` component which should be able to render the correct fonts.
Is this here not enough?

Reply:
Will check that